### PR TITLE
London

### PR DESCRIPTION
**Pull request by iainbeeston**
I hope @cpytel won't mind if I open this pull request for a london guide :smile:

### CODE REVIEW

File: london.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# London
+
+London is a large and diverse city, made up of many different neighborhoods,
+each with someone unique to offer.
```
Comment (cpytel) [inline london.md]: Someone should probably be "something"

**Comment by cpytel** [comment]
I thought this was already merged! Thanks for picking it up! I had one comment, feel free to make that typo correction and merge away!

File: london.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  the first Friday of every month.
+
+## Designers and developers
+
+Some folks to follow:
+
+* [London Dribbblers](http://dribbble.com/designers?location=London)
+* [London Ruby
+  GitHubbers](https://github.com/search?type=Users&language=ruby&q=location:london)
+* [Boston Swift
```
Comment (danbee) [inline london.md]: Should this say "London Swift"?

Comment (iainbeeston) [inline london.md]: Sorry Dan, I merged already (probably a bit hastily judging by feedback).

Can you open a new PR?

Comment (danbee) [inline london.md]: >_<

Comment (iainbeeston) [inline london.md]: @danbee The guidelines for this repo actually say push stuff direct to master. I'll correct this, but for future stuff just go in there and hack away I reckon

**Pull request merged**